usagov/usagov-2021
# USAGOV-1042-published-pages-exporte: Updated Views,config and added 2…

… modules


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1042

## Description


## Type of Changes

- [x ] New Feature
  - Add screenshot showing what it should look like
- [ x] Drupal Config (requires "drush cim")
- [x ] New Modules (requires rebuild)

## Testing Instructions



### Requires New Config

### Requires New Content
- [x ] No

### Validation Steps
url http://localhost/admin/published-pages-report or what ever your localhost is /admin/published-pages-report
Also, at the very bottom of that screen click the csv to export ensure the doc is there and has data.

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
